.github/workflows/code-coverage.yaml

```diff
@@ -0,0 +1,44 @@
+name: PHPUnit Coverage Report
+
+on:
+  push:
+    branches: [develop, master]
+  pull_request:
+    branches: [develop, master]
+  workflow_dispatch: ~
+
+permissions:
+  contents: write
```

Comment:
This means you intend on committing into the git repository in this workflow, I don't think that's necessary.

Reply:
I forgot to delete it

Reply:
Done